index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Twitter. It's what's happening.</title>
+    <link rel="stylesheet" type="text/css" href="./index.css">
+  </head>
+  <body>
+
+    <div id="main">
+
+      <div id="blue">
+          <section>
+              <ul class="blueList">
+                <img id="searchIcon"src="https://img.icons8.com/ios/30/000000/search--v1.png"></img>
+                <li class="blueListItem">Follow your interests.</li>
+                <img id="usersIcon" src="https://img.icons8.com/ios/30/000000/conference-call.png"></img>
+                <li class="blueListItem">Hear what people are talking about.</li>
+                <img id="msgIcon" src="https://img.icons8.com/windows/32/000000/messaging-.png"></img>
+                <li class="blueListItem">Join the conversation.</li>
+              </ul>
+          </section>
+      </div>
+      
+      <div id="white">
+          <section>
+              <form>
+                  <input type="text" class="textInput"id="userInput" placeholder="Phone, email, or username"></input>
+                  <input type="password" class="textInput" id="pwdInput" placeholder="Password"></input>
+                  <input type="button" id="loginBtn1" value="Log in"></input>
+              <form>
+          <section>
+
+          <a id="forgotPwd" href="https://twitter.com/account/begin_password_reset?">Forgot password?</a>
+  
+          <section>
+            <img src="images/birdlogo.png" id="birdLogo"></img>
+            <h1>See what's happening in the world right now.</h1>
+            <h3>Join Twitter today.</h3>
+          </section>
+
+          <section>
+              <button id="signUpBtn">Sign Up</button>
+              <button id="loginBtn2">Login</button>
+          </section>
+      </div>
+  
+    </div>
+
+    <div class="footer">
```

Comment:
Using the <footer> tag (instead of a <div>) would be a better practice. Make sure to be using <header>, <nav>, <main>, <footer>, etc to structure the page (makes it more readable).